src/tls.c

```diff
@@ -1197,7 +1206,7 @@ int TLS_hmac(WOLFSSL* ssl, byte* digest, const byte* in, word32 sz, int padSz,
     }
 #endif
 
-    wolfSSL_SetTlsHmacInner(ssl, myInner, sz, content, verify);
+    wolfSSL_SetTlsHmacInner(ssl, myInner, sz, content, epochOrder);
```

Comment:
How can epochOrder mean the same thing as verify hmac side or create hmac side?

Reply:
This is an incorrect naming in the `wolfSSL_SetTlsHmacInner` function. The `verify` argument is passed directly to `WriteSEQ` as `verifyOrder` and then to `DtlsGetSEQ` as `order` which uses it to determine which epoch and sequence numbers should be used.
The name `verify` makes sense in the TLS (in `GetSEQIncrement`) case since it only determines if we want to verify for sending or receiving. This worked coincidentally for `DtlsGetSEQ` because of how the orders are defined:
```
    PREV_ORDER         = -1,   /* Sequence number is in previous epoch. */
    PEER_ORDER         = 1,    /* Peer sequence number for verify. */
    CUR_ORDER          = 0,    /* Current sequence number. */
```
So if `verify = 1` then this means receiving or `PEER_ORDER` while `verify = 0` means sending or `CUR_ORDER`. This obviously doesn't take the possibility of `PREV_ORDER` into account so I had to modify this part.

Reply:
epochOrder is not the correct name.  epoch does NOT exist in TLS.  We can't rely on side effects / interactions with other items.  That is, if you're trying to combine the verify side and a DTLS order in the same variable, separate them into 2 variables.  Or have a much clearer name, explanation.  Remember, someone has to easily read / maintain / understand each of these items exactly.